tests/neo.UnitTests/SmartContract/UT_JsonSerializer.cs

```diff
@@ -283,7 +283,7 @@ public void Serialize_Array_Bool_Str_Num()
         [TestMethod]
         public void Deserialize_Array_Bool_Str_Num()
         {
-            var items = JsonSerializer.Deserialize(JObject.Parse("[true,\"test\",123]"));
+            var items = JsonSerializer.Deserialize(JObject.Parse("[true,\"dGVzdA==\",123]"));
```

Comment:
I think that also it should work with strings.

Reply:
But how do you distinguish between base64 strings and ordinary strings that only use the base64 character set?

Reply:
> But how do you distinguish between base64 strings and ordinary strings that only use the base64 character set?

Now we can't. But in json it's more common to find a string than a binary content. If we should decide by one, i prefer string. 

Other idea, I think that we should change the deserialization way, and create a object inside the VM. With the type and the value, we had a lot of troubles with json because of this. It's better to deserialize it in `Array(type,value)`, we will be able to deserialize it exactly like it was received.